stable/dex/values.yaml

```diff
@@ -192,6 +192,37 @@ config:
     tlsCert: /etc/dex/tls/grpc/server/tls.crt
     tlsKey: /etc/dex/tls/grpc/server/tls.key
     tlsClientCA: /etc/dex/tls/grpc/ca/tls.crt
+  connectorFiltersHooks:
```

Comment:
Can we add this blocks as commented out?

Reply:
Done in  030920e105693768c5a7cd40cfbe041732f64434